src/ert/config/queue_config.py

```diff
@@ -100,6 +101,14 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
         ):
             _validate_torque_options(queue_options[QueueSystem.TORQUE])
 
+        if (
+            selected_queue_system != QueueSystem.LOCAL
+            and queue_options[selected_queue_system]
+        ):
+            _check_for_overwritten_queue_system_options(
+                selected_queue_system, queue_options[selected_queue_system]
```

Comment:
> I had to modify and complicate this part because I discovered that the option_value might be left unspecified in the config file in which case there is only one value to unpack.
If that is the case, try passing a filter or filter generator to the function.
example
```python
(options_string for opitons_string in queue_options[selected_queue_system] if isinstance(option_string, tuple))
```
This allows you to loop once through the data set instead of twice.
This is a matter of opinion but it seems expensive doing two consecutive loops on the same data set for a log.